openshift/04-deploymentconfig-template.yml

```diff
@@ -98,6 +100,80 @@ objects:
               cpu: 250m
               memory: 250M
           terminationMessagePath: /dev/termination-log
+        - env:
+          - name: RAILS_LOG_TO_STDOUT
+            value: "true"
+          - name: RAILS_ENV
+            value: production
+          - name: DATABASE_URL
+            valueFrom:
+              secretKeyRef:
+                key: DATABASE_URL
+                name: zync
+          - name: SECRET_KEY_BASE
+            valueFrom:
+              secretKeyRef:
+                key: SECRET_KEY_BASE
+                name: zync
+          - name: BUGSNAG_API_KEY
+            valueFrom:
+              secretKeyRef:
+                key: BUGSNAG_API_KEY
+                name: zync
+          - name: SKYLIGHT_AUTHENTICATION
+            valueFrom:
+              secretKeyRef:
+                key: SKYLIGHT_AUTHENTICATION
+                name: zync
+          - name: ZYNC_AUTHENTICATION_TOKEN
+            valueFrom:
+              secretKeyRef:
+                key: ZYNC_AUTHENTICATION_TOKEN
+                name: zync
+          - name: RAILS_LOG_LEVEL
+            value: info
+          args:
+          - que
+          - --worker-count
+          - 10
+          image: zync:latest
+          imagePullPolicy: Always
+          livenessProbe:
```

Comment:
que is the background work processor, it does not start listening on any port

Reply:
Ok Do you think we need some readiness/liveness for this container? If you think so, which command you think would be good for that?

Reply:
I think it is not needed. if the process dies it will kill the container and there is no readiness really. it is all background processing.

Reply:
ok Cool